data/Publications.json

```diff
@@ -3694,5 +3694,41 @@
         "bibcode": "2019AJ....157..101M",
         "doi": "10.3847/1538-3881/aaf88f",
         "description": "Cloud Atlas: HST nir spectral library"
+    },
+    {
+        "name": "Sahl16",
+        "bibcode": "J/MNRAS/455/357",
+        "doi": null,
```

Comment:
These publications sholud have DOIs and descriptions....I'll take a look at how you are adding them.